phase2/ignition/vanilla/kubelet.service

```diff
@@ -15,6 +16,7 @@ ExecStart=/usr/bin/docker run \
         -v /var/run:/var/run:rw \
         -v /var/lib/docker/:/var/lib/docker:rw \
         -v /var/lib/kubelet/:/var/lib/kubelet:shared \
+        -v /var/log:/var/log:shared \
```

Comment:
Here's my question about that: does anything else create the `/var/log/containers` folder?  If we're only sharing `/var/log`, doesn't there stand a chance that we will need to first create the containers folder?  Or, is there another component that will check and create that?